docs/content/en/blog/news/play-environment.md

```diff
@@ -0,0 +1,56 @@
+---
+date: 2022-03-27
+title: "The PipeCD play environment is here!!!"
+linkTitle: "The PipeCD play environment"
+weight: 994
+description: "In this post, we will have a glance at the play environment of PipeCD, how to access and what you can get from the environment."
+author: Khanh Tran ([@khanhtc1202](https://twitter.com/khanhtc1202))
+---
+
+Good news, PipeCD team's happy to bring you a place where you can have a look at the PipeCD platform in use. We call it the `play environment` - [https://play.pipecd.dev](https://play.pipecd.dev).
+
+![play-environment-view](/images/play-environment-overview.png)
+<p style="text-align: center;">
+PipeCD play environment view
+</p>
+
+With this live demo, you can now have a look at the PipeCD platform in use without self preparing [quickstart](/docs/quickstart/) or such. It’s way easier (and faster) to make a try by clicking around and seeing how you can get when using PipeCD as the CD platform on your own.
+
+### How to access
+
+The PipeCD play environment console is available at [https://play.pipecd.dev](https://play.pipecd.dev). After following the link, you will get the login page as below.
+
+![play-environment-login](/images/play-environment-login.png)
+
+Type __play__ to the input box as the project name to login and click to `Continue` to go to the account sign in page. Followed by `LOGIN WITH GITHUB`.
+
+![play-environment-login-github](/images/play-environment-github-login.png)
+
+Then, that's it. Feel free to click around and see what PipeCD can bring to you in real-life use.
+
+#### Some pages you may feel interest
+
+![play-environment-application](/images/play-environment-application.png)
```

Comment:
Can you take another screenshot of this?
The current one is containing many "Terminated" pods since we are using preemptible nodes in the play cluster.

Reply:
Updated https://github.com/pipe-cd/pipecd/pull/3448/commits/3cbecd78b60dd447f99c970b06edc708424b1a8b 🙆‍♂️